src/server/static/staticFiles.ts

```diff
@@ -32,8 +32,7 @@ const cloneCycle = async (props: PropsClone): Promise<void> => {
     // check if sourceDir exists
     try {
       await fs.access(sourceDir)
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    } catch (_e) {
+    } catch (e) {
```

Comment:
shouldnt this give error?

Reply:
I added the rule @sorja

Reply:
No it doesn't work with that rule @sorja , it needs the error rule